infra/cifuzz/cifuzz.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module used by CI tools in order to interact with fuzzers.
+This module helps CI tools do the following:
+  1. Build fuzzers.
+  2. Run fuzzers.
+Eventually it will be used to help CI tools determine which fuzzers to run.
+"""
+
+import logging
+import os
+import shutil
+import sys
+
+import fuzz_target
+
+# pylint: disable=wrong-import-position
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+import build_specified_commit
+import helper
+import repo_manager
+import utils
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    stream=sys.stdout,
+    level=logging.DEBUG)
+
+
+def build_fuzzers(project_name, project_repo_name, commit_sha, git_workspace,
+                  out_dir):
+  """Builds all of the fuzzers for a specific OSS-Fuzz project.
+
+  Args:
+    project_name: The name of the OSS-Fuzz project being built.
+    project_repo_name: The name of the projects repo.
+    commit_sha: The commit SHA to be checked out and fuzzed.
+    git_workspace: The location in the shared volume to store git repos.
+    out_dir: The location in the shared volume to store output artifacts.
+
+  Returns:
+    True if build succeeded or False on failure.
+  """
+  if not os.path.exists(git_workspace) or not os.path.exists(out_dir):
```

Comment:
How about instead of giving an ambiguous error message, why not make these two separate if conditions.

Reply:
Also include the values of the directories in the error message.

Reply:
Done.